scripts/doc-generator.js

```diff
@@ -1,354 +0,0 @@
-#!/usr/bin/env node
-
-'use strict';
-
-require('ts-node/register');
-
-const {
-    TestContext,
-    toString,
-    isString,
-    getTypeOf,
-    firstToUpper,
-    opSchema,
-    isFunction
-} = require('@terascope/job-components');
-
-const generateToc = require('markdown-toc');
```

Comment:
We probably should remove `markdown-toc` and `markdown-table` from the package.json too